### PR TITLE
Add docs on restoring AppInstaller

### PR DESCRIPTION
**Pull request by KevinLaMS**
This is draft.  We need to get URL to older package.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-cli/pull/845)

### CODE REVIEW

File: doc/restore.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+1) Remove your machine from Windows Insider Flights.
+The Windows Package Manager is included in the Windows Insider Flights.  If you are installing a Windows Insider build, you will keep getting updated.
+2) Remove yourself from the Windows Package Manager flights.  
+If you requested to be added to the Windows Package Manager flights, then you will specifically need to ask to be removed.  You can ask to be removed by filling in the following form:
+
+Note: Make sure you use the correct email address.  It must be the address used to sign up for the Insider build.  If your email address is not the same address that you are using to log into the Windows Store, then removing it will have no impact.
+
+Please allow for 1 week, to be sure the form has been processed, though most likely it is processed much sooner.
+
+3) Install an older build of the [Desktop App Installer](https://www.microsoft.com/en-us/p/app-installer/9nblggh4nns1).  We have placed an earlier released version [here](https://www.github.com/micrsoft/winget-cli/releases).
```
Comment (denelon) [inline doc/restore.md:14]: we should remove /en-us/ as the site handles localization.

**Comment by jsoref** [comment]
Can you please rebase? It'll make the bot much happier.

File: doc/restore.md (hunk 1 above)
Comment (lechacon) [inline doc/restore.md:14]: I think we don't want to use "here" for links? I read somewhere that it makes it harder to parse what the link refers to, both for people and tools like screen readers.

Maybe something like `Here is an [earlier released version for download](...).`

Comment (jsoref) [reply doc/restore.md:14]: I'd probably write `There is...` or `An [earlier version(...) is available for download.`

And yes, avoid linking `here` it's screen reader hostile.

**Comment by JohnMcPMS** [comment]
I think the ship has sailed on this PR, as well as going back to older versions generally.

**Pull request closed without merging**